### DISCUSSION TITLE
Background navigation / screen off tracking

### DISCUSSION THREAD

**Discussion by 1Metrodriver** [osmandapp/OsmAnd#15493] [Maps Related]
I just upgraded to the last version of Osmand, and the background navigation function that this app has had since I first got it many years ago (2007?) is gone. I use the app mostly for hiking (all over the world) and tracking my progress with the screen off to save battery power (important out in the wild). Now the tracking stops when I turn off the screen with the power button, and the entire app gets killed when I open a different app. This has made Osmand useless for me

**Comment by ThomasMeister123** [comment]
I have the latest Version too. Here everything works fine: turn off Display, open other app,.. Navigation goes on in background including Navigation-voice

**Comment by tommycrock** [comment]
It might be a power saving setting in Android

**Comment by 1Metrodriver** [comment]
I checked all my settings, including the app settings, android settings and Location settings.
I tried again a walk, with Prevent Standalone Logging enabled and disabled, that made no difference. As soon as I turn the screen off, tracking stops and restarts when I open the phone again. Using another app kills Osmand, it has to start up again and defaults to the world map. Several versions ago this worked great, used it like that since I first got it in the early days of Osmand.
How can I go back to an earlier version of Osmand? I cannot go on a hike with the screen on and run out of battery power halfway.
My phone is a Samsung S20 5G, Osmand+ 4.2.7

**Comment by scaidermern** [comment]
Did you actually check whether _battery optimization_ for OsmAnd is set to _off_ in your Android settings?

Also refer to the following documentation for more information:
* https://osmand.net/docs/user/troubleshooting/track-recording-issues/#the-system-may-kill-background-apps-to-save-power
* https://dontkillmyapp.com/samsung

**Reply by 1Metrodriver** [reply]
Yesterday I checked the Playstore for an update for Osmand, which was not there. There was however the option to turn on Beta testing, which I did. The background nav worked as before, with phone screen turned off, or during use of different apps or phone calls.
